canonical-web-and-design/maas-ui
# Upgrade to Formik 2.0.3

Done:
- Upgrade to Formik 2.0.3. Fixes: #386.
- Update fields to use useField instead of passing formikProps.
- Update forms to use useFormikContext instead of passing formikProps.
- Remove nested form field components where possible.

QA:
- Every form has been updated so starting at the login make sure they all work. Sorry!
